### PR TITLE
EPMRPP-48945: [CORE] [UI] Admin. Plugin. Add confirmation pop up on disable/enable action for toggler

### PR DESCRIPTION
**Pull request by KatsiarynaTatarynovich**

### CODE REVIEW

File: app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx [hunk 1]
```diff
@@ -112,6 +133,47 @@ export class InstalledTab extends Component {
...
+  showEnablePluginModal = (pluginName, callback) => {
+    const {
+      intl: { formatMessage },
+    } = this.props;
+
+    this.props.showModalAction({
+      id: 'confirmationModal',
+      data: {
+        message: formatMessage(messages.enablePluginMessage, { pluginName }),
+        onConfirm: () => callback(),
```
Comment (AmsterGet) [inline app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx:145]: Just `callback`

File: app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx [hunk 2]
```diff
@@ -112,6 +133,47 @@ export class InstalledTab extends Component {
...
+  showDisablePluginModal = (pluginName, callback) => {
+    const {
+      intl: { formatMessage },
+    } = this.props;
+
+    this.props.showModalAction({
+      id: 'confirmationModal',
+      data: {
+        message: formatMessage(messages.disablePluginMessage, { pluginName }),
+        onConfirm: () => callback(),
```
Comment (AmsterGet) [inline app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx:162]: Same here

File: app/src/pages/admin/pluginsPage/pluginsListItems/pluginsItem/pluginsItem.jsx [hunk 3]
```diff
@@ -132,7 +134,10 @@ export class PluginsItem extends Component {
         <div className={cx('plugins-additional-block')}>
           {toggleable && (
             <div className={cx('plugins-switcher')} onClick={(e) => e.stopPropagation()}>
-              <InputSwitcher value={this.state.isEnabled} onChange={this.onToggleActiveHandler} />
+              <InputSwitcher
```
Comment (AmsterGet) [inline app/src/pages/admin/pluginsPage/pluginsListItems/pluginsItem/pluginsItem.jsx:137]: According to the task description: `Disable/enable switcher should be removed form the main plugin tab`.

File: app/src/components/integrations/containers/integrationInfoContainer/infoSection/infoSection.jsx [hunk 4]
```diff
@@ -139,7 +141,12 @@ export class InfoSection extends Component {
                       })
                 }
               >
-                <InputSwitcher value={isEnabled} onChange={this.onToggleActiveHandler} />
+                <InputSwitcher
+                  value={isEnabled}
+                  onChange={() =>
+                    showPluginModal(isEnabled, title || name, this.toggleActiveHandler)
```
Comment (AmsterGet) [inline app/src/components/integrations/containers/integrationInfoContainer/infoSection/infoSection.jsx:147]: I think will be better to edit `this.toggleActiveHandler` function to handle this case.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/pages/admin/pluginsPage/pluginsListItems/pluginsItem/pluginsItem.jsx (hunk 3 above)
Comment (KatsiarynaTatarynovich) [reply app/src/pages/admin/pluginsPage/pluginsListItems/pluginsItem/pluginsItem.jsx:137]: Please, see the comment https://jiraeu.epam.com/browse/EPMRPP-48945?focusedCommentId=6695048&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6695048

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**